python/src/aiconfig/editor/client/src/components/prompt/AddPromptButton.tsx

```diff
@@ -21,8 +21,8 @@ const useStyles = createStyles((theme) => ({
     borderRadius: rem(4),
     display: "flex",
     justifyContent: "center",
+    marginLeft: "36px",
```

Comment:
nit: Btw why 36px in particular? If this is used multiple places, should this be defined as a STYLE constant somewhere?

Reply:
I'm not sure exactly what is making it add up to that -- the cell menu button padding  icon is what pushes the prompt cell over to require that alignment from the other components. Good call on using a const - will do